test/functional/VM_Test/build.xml

```diff
@@ -131,7 +131,9 @@
 	<target name="clean" depends="dist" description="clean up">
 		<!-- Delete the ${build} directory trees -->
 		<delete dir="${build}" />
-		<delete dir="${data}/*.class" />
+		<delete>
+			<fileset dir="${data}" includes="**/*.class" />
```

Comment:
This deletes all class files in the data directory and in all subdirectories under data.  Is that intentional?  The commit comment implies otherwise

Reply:
Defer to test team. I assumed they meant to delete all class files in data dir. Not sure about subdirs.

Reply:
@renfeiw ^

Reply:
Yes, I believe the class files need to be deleted after they get jared into VM_TEST.jar. The changeset looks good to me, could you please run a sanity build to make sure.

Reply:
> I'm not sure Renfei answered your question. 

Agreed.  @renfeiw did you want just the class files in the `data/` directory to be deleted or did you want class files in the subdirectories to be deleted as well?

Reply:
Based on the comment and the activity in the "dist" target of the ant script, it is right to assume that clean up means all class files in data and its subdirs.

Reply:
Yes, we need to clean up all class files.